operator/example-cassdc-yaml/dse-6.8.x/example-cassdc-minimal.yaml

```diff
@@ -23,6 +23,7 @@ spec:
     jvm-server-options:
       initial_heap_size: "800M"
       max_heap_size: "800M"
+      max_direct_memory: "800M"
```

Comment:
What's the point here? The default from modern JDK should be the same?

Reply:
no - this is about the cassandra startup scripts calculating a poor value inside the container

Reply:
Is there a reason we wouldn't fix such calculations in cass-operator?